cmd/spicedb/serve.go

```diff
@@ -106,6 +111,7 @@ func registerServeCmd(rootCmd *cobra.Command) {
 	// Flags for configuring dispatch behavior
 	serveCmd.Flags().Uint32("dispatch-max-depth", 50, "maximum recursion depth for nested calls")
 	cobrautil.RegisterGrpcServerFlags(serveCmd.Flags(), "dispatch-cluster", "dispatch", ":50053", false)
+	serveCmd.Flags().String("dispatch-upstream-addr", "", "upstream grpc address to dispatch to")
```

Comment:
I think we should just drop everything servok and see what the minimal set of flags would be.

Reply:
did it in a separate commit in case we want to revert